src/AdminOps.ts

```diff
@@ -31,7 +29,7 @@ export const addToMap = async ({
     )
   }
 
-  const user = await User.findByUsername({ username })
+  const user = await User.getUser({ username })
```

Comment:
we have other way to get the user. by phone number for instance. does this name makes sense as a result?